modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/categorization/EditCategories.es.js

```diff
@@ -12,15 +12,14 @@
  * details.
  */
 
-/* eslint no-unused-vars: "warn" */
-
+import 'asset-taglib/asset_categories_selector/AssetCategoriesSelector.es';
 import 'clay-multi-select';
 import 'clay-radio';
+import 'frontend-js-web/liferay/compat/modal/Modal.es';
 import Component from 'metal-component';
-import {Config} from 'metal-state';
 import Soy from 'metal-soy';
-import {Modal} from 'frontend-js-web';
-import 'asset-taglib/asset_categories_selector/AssetCategoriesSelector.es';
+import {Config} from 'metal-state';
+
```

Comment:
1. We should sort parameters by **module** name, the thing on the right.
1. `AssetCategoriesSelector` and `Modal` should be in the block with internal imports.